maxim/logger/portkey/portkey.py

```
@@ -0,0 +1,184 @@
+            )
+
+        # Call the actual Portkey completion
+        response = self._client.chat.completions.create(*args, **kwargs)
+
+        try:
+            if generation is not None:
+                generation.result(OpenAIUtils.parse_completion(response))
+            if is_local_trace and trace is not None:
+                trace.set_output(response.choices[0].message.content or "")
+                trace.end()
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)